ssaHAL/examples/leds.py

```
@@ -0,0 +1,77 @@
+from ssa import SSA, ssa_main
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)